src/snowflake/telemetry/_internal/exporter/otlp/proto/metrics_exporter/__init__.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
+#
+
+import abc
+import enum
+
+from opentelemetry.sdk.metrics.export import (
+    MetricExportResult,
+    MetricExporter,
+    MetricsData
+)
+from snowflake.telemetry._internal.encoder.otlp.proto.common.metrics_encoder import (
+    serialize_metrics_data
+)
+
+
+class MetricWriterResult(enum.Enum):
```

Comment:
Same comments: update comment, can we just make this an "alias" of the `MetricWriterResult` or even make the writer return `MetricWriterResult`?

Reply:
Since MetricExporterResult is not in an experimental module, I changed write_metrics so it just returns MetricExporterResult.